src/main/java/org/futurecollars/lesson8task1names/NamesListCreator.java

```diff
@@ -0,0 +1,28 @@
+package org.futurecollars.lesson8task1names;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NamesListCreator {
```

Comment:
Ta klasa to trochę przerost formy nad treścią. Nic nie daje, a jedynie opakowuje listę.